wp-content/themes/workingnyc/search.php

```diff
@@ -35,9 +32,9 @@
   } else {
     return new WorkingNYC\Jobs($p);
   }
-}, $job_or_program_posts);
+}, $posts);
 // $context['language'] = ICL_LANGUAGE_CODE;
 
 // Render view
-$templates = array('search.twig');
+$templates = array('search/search-results.twig');
```

Comment:
All other twig files related to search were already in the `search` folder, so I moved this file as well